metadetect/lsst_measure.py

```diff
@@ -23,50 +22,104 @@
 
 from . import util
 from . import procflags
-from .lsst_mbobs_extractor import MBObsMissingDataError
+from .defaults import DEFAULT_LOGLEVEL
 
+# this is good for ksigma
+DEFAULT_STAMP_SIZE = 64
 
-def measure_weighted_moments(
-    mbobs,
-    weight,
-    thresh=10,
-    loglevel='INFO',
+# threshold in units of the standard deviation of the sky noise
+DEFAULT_THRESH = 5.0
+
+
+def detect_and_measure(
```

Comment:
I wonder if this function should be `detect_deblend_and_measure`?

Reply:
In the stack deblending is always part of the hierarchical detection method; but one has the choice whether to do followup measurements on deblended stamps.

Reply:
Actually I think that's not true that deblending is always part of detection

Reply:
I'll rename